### PR TITLE
Rename stage names

### PR DESCRIPTION
**Pull request by wolfs**
The stage names have been causing some confusion. This PR renames the stages so that their purpose is easier to understand.

### CODE REVIEW

File: .teamcity/Gradle_Check/model/CIBuildModel.kt [hunk 1]
```diff
@@ -8,6 +8,18 @@ import configurations.SanityCheck
 import configurations.SmokeTests
 import jetbrains.buildServer.configs.kotlin.v2018_1.BuildType
 
+
+enum class StageNames(override val stageName: String, override val description: String) : StageName{
+    QUICK_FEEDBACK_LINUX_ONLY("Quick Feedback - Linux Only", "Run checks and functional tests (embedded executer)"),
+    QUICK_FEEDBACK("Quick Feedback", "Run checks and functional tests (embedded executer)"),
```
Comment (lptr) [inline .teamcity/Gradle_Check/model/CIBuildModel.kt]: Add the OSs to the description?

Comment (wolfs) [reply .teamcity/Gradle_Check/model/CIBuildModel.kt]: Better now?

File: .teamcityTest/Gradle_Check_Tests/CIConfigIntegrationTests.kt [hunk 2]
```diff
@@ -30,7 +32,7 @@ class CIConfigIntegrationTests {
     fun macOSBuildsSubset() {
         val m = CIBuildModel()
         val p = RootProject(m)
-        val releaseAccept = p.subProjects.find { it.name.contains("Release Accept") }!!
+        val releaseAccept = p.subProjects.find { it.name.contains("Ready for Release") }!!
```
Comment (blindpirate) [inline .teamcityTest/Gradle_Check_Tests/CIConfigIntegrationTests.kt]: Why not `READY_FOR_RELEASE` here?

Comment (wolfs) [reply .teamcityTest/Gradle_Check_Tests/CIConfigIntegrationTests.kt]: Better now?

Comment (wolfs) [reply .teamcityTest/Gradle_Check_Tests/CIConfigIntegrationTests.kt]: Yeah, I realized that. That is the current code in this PR (I force pushed):
```
 val readyForRelease = p.subProjects.find { it.name.contains(StageNames.READY_FOR_RELEASE.stageName) }!!
```

Is that good enough? Or did you expect something else?

**Pull request merged**